server/views/components/interview/index.njk

```diff
@@ -0,0 +1,12 @@
+<div class="interview">
+  {% for qAndA in interview %}
+    <p class="interview__question">
+      <span class="interview__participant">{{ qAndA.question.participant }}</span>
```

Comment:
I reckon we stick with the spec recommendation. And the `span`s should stay because block level elements aren't supposed to be inside `p`s.